### PR TITLE
chore: use theming-base-content for --sap* CSS vars

### PR DESCRIPTION
**Pull request by ilhan007**
We used to provide a subset of all --sap* variables (the ones our components need), provided by `@sap-theming/theming-base-content` project and respectively generated from the `ThemeDesigner.` 
However, our users might need some of these variables we don't use and provide and **we agreed on switching to `@sap-theming/theming-base-content` as a source for all theme parameters** and thus having all the parameters available for the users of the`@ui5/webcomponents`

- use  the external `theming-base-content` npm package as a source for all base parameters
- remove all the `base-parameter.css` files we used to have
- remove the `less` dependency (as there is nothing to calculate any more)  from `theme-base`package
- remove the `rollup-plugin-less` dependency from `tools` package
- remove the `watch` task for `theme-base`
- strip out unneeded classes and CSS vars:
-- classes: `.background-image*`, `.sapThemeMeta*`
-- vars: `.sapContrast :root{--sap*}` and `--sapFontUrl*`
- add manually  the `--sapSubtleLinkColor` variable as not yet available in `theming-base-content`
- the size (transferred over the network)  of the `parameters-bundle.css.json` increases `from 1.9kb to 3kb` (resource size from 8.9 kb to 14,8kb)
- the size of the bundle, when the theme is inlined, increases with additional `+2kb`

FIXES: https://github.com/SAP/ui5-webcomponents/issues/1396

### CODE REVIEW

File: packages/tools/lib/postcss-css-to-esm/index.js [hunk 1]
```diff
@@ -19,15 +19,21 @@ module.exports = postcss.plugin('add css to esm transform plugin', function (opt
...
 
 	return function (root) {
-		const css = JSON.stringify(root.toString());
-		const targetFile = root.source.input.from.replace(`/${opts.toReplace}/`, "/dist/generated/").replace(`\\${opts.toReplace}\\`, "\\dist\\generated\\");
+		const r = new RegExp(/[\s\S]*(:root{[\s\S]*})/, 'g');
+		let css = root.toString();
+
+		const match = r.exec(css);
+		if (match) {
+			css = match[1];
```
[the remark this answers is not in the document]
Comment (ilhan007) [reply packages/tools/lib/postcss-css-to-esm/index.js]: This part is not really ready - to strip all unneeded parts from theming-base-content, but yes you are right it is possible

File: packages/theme-base/src/themes/sap_belize/base-parameters-vars.less [hunk 2]
```diff
@@ -1,297 +1,5 @@
-@import "base-parameters.less";
+@import "../../../../../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_belize/css_variables.less";
```
[the remark this answers is not in the document]
Comment (ilhan007) [reply packages/theme-base/src/themes/sap_belize/base-parameters-vars.less]: after the next commit we no longer use lessc, so this is outdated

File: package.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
     "prepare:fiori": "cd packages/fiori && nps prepare",
     "dev:base": "cd packages/base && nps watch",
     "dev:localization": "cd packages/localization && nps watch",
-    "dev:theme-base": "cd packages/theme-base && nps watch",
+    "dev:theme-base": "cd packages/theme-base",
```
Comment (vladitasev) [inline package.json:32]: The whole "dev:theme-base" command can be deleted both here and where it is called (in "start: all")

**Comment by vladitasev** [comment]
@dimovpetar FYI

**Pull request merged**